.travis.yml

```diff
@@ -39,17 +39,12 @@ jobs:
       arch: amd64
       before_script:
         - sudo apt-get install upx-ucl -y
-        - sudo apt-get install gcc-aarch64-linux-gnu -y
-        - sudo apt-get install libc6-dev-arm64-cross -y
-        - sudo apt-get install gcc-arm-linux-gnueabi -y
-        - sudo apt-get install libc6-dev-armel-cross -y
         - export GOFLAGS=-mod=vendor
       script:
         - make
         - make bluetoothdevice
         - make smallbuild
-        - make crossbuild
-        - make crossbuild GOARM=GOARM7
```

Comment:
crossbuild should not be deleted, see the name of job @Line38

Reply:
Smallbuild havs cover all logics that crossbuild does, why need to re-test them?

Reply:
Smallbuild is used to compresses the binary(for amd64 and arm) and crossbuild is used to build for arm, they should be different.

Reply:
Why need to add limit for arch? We can compress the binary for all arches(amd64, arm, arm64).